src/webpack/index.ts

```diff
@@ -141,10 +155,10 @@ export function getWebpackPlugin<UserOptions = {}> (
         }
 
         // load hook
-        if (plugin.load && plugin.__vfsModules) {
+        if (plugin.load) {
           compiler.options.module.rules.push({
-            include (id) {
-              return id != null && plugin.__vfsModules!.has(id)
+            include () {
+              return true
```

Comment:
Can we have an internal map for the encodedId, so the hooks do not apply to every module?

Reply:
What I was trying to fix with this is that the load hook wasn't called for every module (like it is in rollup - at least if I follow the diagram [here](https://rollupjs.org/guide/en/#build-hooks) correctly).

Are you suggesting we change it so it doesn't run when the module is marked as external? If so, I just pushed that behavior along with some tests: ee12eb7

Let me know what you think!

Reply:
I guess you are right. I originally thought the `load` hook would only be called for the ids returned by `resolveId` - probably a myth from nowhere that has been spinning in my mind for a long time. 🤣